### ISSUE TITLE
Extension Host Terminated unexpectedly

### ISSUE THREAD

**Issue by shaft49** [microsoft/vscode#105848]
- VSCode Version: 1.48.0
- OS Version: macOS 10.15.6

Steps to Reproduce:

1.  No matter how I open vscode, a pop up is shown at the beginning.

Does this issue occur when all extensions are disabled?: Yes

I've disabled all the extensions and even installed freshly but the issue does not go away.

**Comment by alexdima** [comment]
@shaft49 Can you please also try the insiders version from https://code.visualstudio.com/insiders/ . In the insiders version, I have added some more troubleshooting information to the developer console when the extension host crashes.

**Comment by shaft49** [comment]
@alexdima I've tried the insiders version, but the same issue persists.

**Comment by alexdima** [comment]
@shaft49 When the extension host crashes, can you please click "Open Developer Tools" and paste here the output:

e.g.
```
Extension host terminated unexpectedly. Code:  null  Signal:  SIGTERM
Extension host terminated unexpectedly. The following extensions were running: vscode.microsoft-authentication, vscode.github-authentication, vscode.git
```

**Comment by shaft49** [comment]
<img width="549" alt="Screenshot 2020-09-03 at 9 32 36 AM" src="https://user-images.githubusercontent.com/20694116/92068401-9e1a7e80-edc8-11ea-9e4b-82c446a40a4f.png">
Here's the screenshot of the console log.

**Comment by alexdima** [comment]
@shaft49 
- Have you by any chance customized your `DYLD_LIBRARY_PATH`?
- Does the extension host crash when launching code from the command line?
- Our [Insiders version](https://code.visualstudio.com/insiders/) upgrades Electron from version 7 to 9, and that might bring more stability on some systems, could you please try that ?

This looks similar to #104525 and #103057 .

**Comment by shaft49** [comment]
Yes, DYLD_LIBRARY_PATH was customized to mysql. After deleting that line from .zshrc file everything is working without showing the extension host terminated pop up.

**Comment by alexdima** [comment]
I will mark it as a duplicate of #104525

FYI @bpasero this is the same `DYLD_LIBRARY_PATH` issue from #104525 . In the fix for #104525 , I have removed the environment variable from the extension host spawning, but if you take a quick look at the screenshot above, you can see that the file watcher crashes as well.

**Comment by bpasero** [comment]
@alexdima maybe your fix needs to be applied to all the places where we spawn child processes, e.g. everywhere we use `bootstrap-fork` at least.
